src/public/audits-table.vue

```diff
@@ -169,7 +259,13 @@ export default {
             table_height: 769,
             extra_height: 0,
             source: null,
-            filter: {}
+            filter: {
+                user_id: '',
+                from: '',
+                to: '',
+            },
+            type_filter: '',
```

Comment:
Is there a reason why the `type_filter` is not a part of `filter` as the user and the date range? (also, it may be clearer to call the attribute `filters` in plural).